specification/serialconsole/resource-manager/Microsoft.SerialConsole/stable/2018-05-01/examples/GetConsoleDisabledSuccessExample.json

```diff
@@ -0,0 +1,14 @@
+{
+    "parameters": {
+        "api-version": "2018-05-01",
+        "subscriptionId": "00000000-00000-0000-0000-000000000000",
+        "default": "default"
+    },
+    "responses": {
+        "200": {
+            "body": {
+                "disabled": true
+            }
+        }
```

Comment:
@asinn826 I know you do have another example for the same operation which covers `"default"` response. Sadly, currently, our tool requires that all response codes have to be in each example. I would propose to merge the two success and error examples to one file.